packages/frontend/src/components/common/DataExplorer/MapViz.tsx

```diff
@@ -101,7 +102,7 @@ export function MapViz({ dataframe }: Props): JSX.Element {
         accessToken: process.env.REACT_APP_SCOUT_MAPBOX_API_KEY ?? '',
         container: 'map',
         style: 'mapbox://styles/mapbox/light-v11',
-        center: [-73.95, 40.72],
+        center: coordinate[city as keyof typeof coordinate],
```

Comment:
If `city` does not exist in `coordinate` this returns `undefined` which causes Mapbox to error. We should still have a default value for that case. A common solution would be to default to `[0, 0]` and change the `zoom` value to zoom out and show the whole map - this is the most common approach to showing an 'empty' map.